src/components/v5/common/ActionSidebar/partials/AmountField/partials/TokenList.tsx

```diff
@@ -0,0 +1,51 @@
+import { Id } from '@colony/colony-js';
+import React, { type FC } from 'react';
+
+import { type ColonyBalancesFragment } from '~gql';
+import { type Token } from '~types/graphql.ts';
+import { getBalanceForTokenAndDomain } from '~utils/tokens.ts';
+import HoverWidthWrapper from '~v5/shared/HoverWidthWrapper/HoverWidthWrapper.tsx';
+
+import { TokenItem } from './TokenItem.tsx';
+
+interface TokenListProps {
+  tokens: Token[];
+  domainId?: number;
+  balances?: ColonyBalancesFragment | null;
+  onSelect: (selectedTokenAddress: string) => void;
+}
+
+export const TokenList: FC<TokenListProps> = ({
+  tokens: colonyTokens,
+  domainId,
+  balances: colonyBalances,
+  onSelect,
+}) => {
+  return (
+    <ul>
+      {colonyTokens.map((colonyToken) => {
+        const tokenBalance = getBalanceForTokenAndDomain(
+          colonyBalances,
+          colonyToken.tokenAddress,
+          domainId ?? Id.RootDomain,
+        );
+
+        return (
+          <li key={colonyToken.tokenAddress} className="mb-1 last:mb-0">
```

Comment:
I know you just ported this over so **this is not a change request** and something I just saw @mmioana but I think it will be neater if we removed the styles for `li` and applied `className="space-y-1"` to the `ul` element instead like:

```js
<ul className="space-y-1">
  <li />
</ul>
```

It should yield the same outcome 👌